admittance_controller/src/admittance_rule.cpp

```diff
@@ -430,8 +443,12 @@ void AdmittanceRule::calculate_admittance_rule(
       desired_acceleration_previous_arr_[i] = acceleration;
       desired_velocity_previous_arr_[i] = desired_velocity_arr_[i];
 
-//       RCLCPP_INFO(rclcpp::get_logger("AR"), "%e = %e - (1/M(%.1f)) (%e - D(%.1f)*%e - S(%.1f)*%e)",
-//                   acceleration, feedforward_acceleration[i], mass_[i], measured_wrench[i], damping_[i], desired_velocity_arr_[i], stiffness_[i], pose_error[i]);
+//       RCLCPP_INFO(rclcpp::get_logger("AR"), "Adm. Rule [%zu]: (%e = %e - D(%.1f)*%e - S(%.1f)*%e)", i,
+//                   acceleration, measured_wrench[i], damping_[i], desired_velocity_arr_[i],
+//                   stiffness_[i], pose_error[i]);
+
+//       RCLCPP_INFO(rclcpp::get_logger("AR"), "Adm. acc [%zu]: %e = %e - (1/M(%.1f)) (%e - D(%.1f)*%e - S(%.1f)*%e)",
+//                   i, acceleration, feedforward_acceleration[i], mass_[i], measured_wrench[i], damping_[i], desired_velocity_arr_[i], stiffness_[i], pose_error[i]);
```

Comment:
consider deleting this debug code, if you're done with it

Reply:
removed and added output to the state message.